presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterStats.java

```diff
@@ -13,37 +13,39 @@
  */
 package com.facebook.presto.orc;
 
+import com.facebook.presto.orc.metadata.StripeInformation;
 import org.weakref.jmx.Managed;
 import org.weakref.jmx.Nested;
 
 import java.util.concurrent.atomic.AtomicLong;
 
-import static com.facebook.presto.orc.OrcWriterStats.FlushReason.CLOSED;
-import static com.facebook.presto.orc.OrcWriterStats.FlushReason.DICTIONARY_FULL;
-import static com.facebook.presto.orc.OrcWriterStats.FlushReason.MAX_BYTES;
-import static com.facebook.presto.orc.OrcWriterStats.FlushReason.MAX_ROWS;
+import static com.facebook.presto.orc.WriterStats.FlushReason.CLOSED;
+import static com.facebook.presto.orc.WriterStats.FlushReason.DICTIONARY_FULL;
+import static com.facebook.presto.orc.WriterStats.FlushReason.MAX_BYTES;
+import static com.facebook.presto.orc.WriterStats.FlushReason.MAX_ROWS;
 import static com.google.common.base.MoreObjects.toStringHelper;
 
 public class OrcWriterStats
+        implements WriterStats
 {
-    public enum FlushReason
-    {
-        MAX_ROWS, MAX_BYTES, DICTIONARY_FULL, CLOSED
-    }
-
     private final OrcWriterFlushStats allFlush = new OrcWriterFlushStats("ALL");
     private final OrcWriterFlushStats maxRowsFlush = new OrcWriterFlushStats(MAX_ROWS.name());
     private final OrcWriterFlushStats maxBytesFlush = new OrcWriterFlushStats(MAX_BYTES.name());
     private final OrcWriterFlushStats dictionaryFullFlush = new OrcWriterFlushStats(DICTIONARY_FULL.name());
     private final OrcWriterFlushStats closedFlush = new OrcWriterFlushStats(CLOSED.name());
     private final AtomicLong writerSizeInBytes = new AtomicLong();
 
-    public void recordStripeWritten(FlushReason flushReason, long stripeBytes, int stripeRows, int dictionaryBytes)
+    @Override
+    public void recordStripeWritten(int stripeMinBytes, int stripeMaxBytes, int dictionaryMaxMemoryBytes, FlushReason flushReason,
+            int dictionaryBytes, StripeInformation stripeInformation)
```

Comment:
Ya, we usually don't trust IDE auto formatting. The coding convention is to either 
- have one parameter per line if the putting into a single line is too long, or
- have everything in one line if the line is short.

So in this case, we will have
```java
public void recordStripeWritten(
    int stripeMinBytes,
    int stripeMaxBytes,
    int dictionaryMaxMemoryBytes,
    FlushReason flushReason,
    int dictionaryBytes,
    StripeInformation stripeInformation)
{
...
}

Reply:
Fixed